script/utils/DeployHelper.sol

```diff
@@ -894,6 +942,8 @@ contract DeployHelper is
 
         // add evenSplitGroupPool to whitelist of group pools
         groupingModule.whitelistGroupRewardPool(address(evenSplitGroupPool), true);
+
+        disputeModule.whitelistDisputeTag("PLAGIARISM", true);
```

Comment:
Why do we need to whitelist dispute tag here?

Reply:
It's for the testing that tokenizer module will revert if the IP is tagged. Moved this line to the test function instead in [f90eb33](https://github.com/storyprotocol/protocol-periphery-v1/pull/144/commits/f90eb3335d7e1c5e9138b4fd4e0b7ffe1786fc18) for better readability :)